Containerfile.alpine

```
@@ -7,7 +7,7 @@ COPY . .
 RUN cargo build --release --bin mollysocket
 
 
-FROM alpine:3 AS runtime
+FROM alpine AS runtime
```

```suggestion
FROM alpine:latest AS runtime
```

Isn't it better to set the latest tag ?

---

It doesn't matter for alpine but sure